libraries/ms-common/src/stm32f0xx/gpio.c

```diff
@@ -3,6 +3,7 @@
 #include <stdbool.h>
 #include <stdint.h>
 
+#include "gpio_cfg.h"
 #include "status.h"
 #include "stm32f0xx.h"
 #include "stm32f0xx_gpio.h"
```

Comment:
Noted, as mentioned before I like to be explicit that I am mainly using _gpio.h stm32f0xx.h is only necessary because it is where ENABLE is defined.